src/App.vue

```diff
@@ -2,7 +2,10 @@
   <div id="app">
     <img alt="Vue logo" src="./assets/logo.png" />
     <HelloWorld msg="Hello Vue 2 + Vite" />
-    <Counter/>
+
+    <h1>App Component: {{ applicationNumber }}</h1>
+    <button @click="applicationNumber+=10">Increment Ten From Parent Cmponent</button>
+    <Counter v-model="applicationNumber" @increment="applicationNumber++" @decrement="applicationNumber--" />
```

Comment:
@DaudIbrahim 

- You should handle increase and decrease logic inside counter component, additional events should not be used to accomplish this outside of counter component since you loose code reusability. 
- Everything (increment, decrement and typing value by keyboard) should be unified with v-model (aka combination of value prop and input event)